docs/dialect/ops_reference.rst

```diff
@@ -262,7 +263,7 @@ following:
 
 Note that the application only takes place for elements that are present in the
 matrix. Thus, the operation will not apply when the values are missing in the
-tensor. For example, `[0.0, 1.0, 0.0] / 2.0 == [0.0, 0.5, 0.0]`.
+tensor. For example, `1.0 / [0.0, 2.0, 0.0] == [0.0, 0.5, 0.0]`.
```

Comment:
Let's avoid representing missing values as zeros. Maybe use a dash? `1.0 / [-, 2.0, -] == [-, 0.5, -]`

Reply:
I used underscores in https://github.com/metagraph-dev/mlir-graphblas/commit/35a3d57cb129d01978886379d0c1aab19388680d.